docs/WalletGuide.md

```diff
@@ -699,3 +707,224 @@ suspend fun main() {
+val second = RecoveryServerKey("second")
+val firstServer = RecoveryServer("recovery.example.com", "auth.example.com", "example.com")
+val secondServer = RecoveryServer("recovery2.example.com", "auth2.example.com", "example.com")
+val servers = mapOf(first to firstServer, second to secondServer)
+val recovery = wallet.recovery(servers)
+val sponsor = SigningKeyPair.fromSecret("<example key>")
+-->
+
+### Create recoverable account
+
+<!--- INCLUDE 
+suspend fun createAccount() {
+-->
+
+<!--- SUFFIX 
+}
+-->
+
+First, let's create an account key and a device key that will be attached to this account
+```kotlin
+  val account = wallet.stellar().account().createKeyPair()
+  val deviceKey = wallet.stellar().account().createKeyPair()
+```
+
+Next, we need to define SEP-30 identities. In this example we are going to use the same list of identities on both recovery servers. 
+```kotlin
+  val recoveryKey = wallet.stellar().account().createKeyPair()
+
+  val identities =
+    listOf(
+      RecoveryAccountIdentity(
+        RecoveryRole.OWNER,
+        listOf(RecoveryAccountAuthMethod(RecoveryType.STELLAR_ADDRESS, recoveryKey.address))
+      )
+    )
+```
+Here, stellar key will be used as a recovery method. Other recovery servers may support email or phone as a recovery methods
+
+You can read more about SEP-30 identities [here](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0030.md#common-request-fields)
+
+Next, let's create a recoverable account:
+```kotlin
+  val recoverableWallet =
+    recovery.createRecoverableWallet(
+      RecoverableWalletConfig(
+        account,
+        deviceKey,
+        AccountThreshold(10, 10, 10),
+        mapOf(first to identities, second to identities),
+        SignerWeight(10, 5),
+        sponsor
+      )
+    )
+```
+With given parameters, this function will:
+1. Create `account` on the Stellar network (if it's not created yet).
+2. `sponsor` will be used to sponsor all operations. You may not sponsor creating recoverable wallet, but in that case `account` must already be created.
+3. `deviceKey` will be used as a primary account key. Please note that master key belonging to `account` will be locked. `deviceKey` should be used as a primary signer instead.
+4. Set all account thresholds to 10. You can read more about threshold in the [documentation](https://developers.stellar.org/docs/encyclopedia/signatures-multisig#thresholds)
+5. Use identities that were defined earlier on both servers. (That means, both server will accept SEP-10 authentication via `recoveryKey` as an auth method)
+6. Set device key weight to 10, and recovery server weight to 5. Given account thresholds, both servers must be used to recover account, as transaction signed by one will only have weight of 5, which is not sufficient to change account key.
+
+Finally, sign and submit transaction to the network
+```kotlin
+  val tx = recoverableWallet.transaction.sign(account).sign(sponsor)
```

Comment:
Q: could we change the `sign()` function to take in a variable number of arguments?
```suggestion
  val tx = recoverableWallet.transaction.sign(account, sponsnor)
```

Reply:
If we change it to vararg, it will accept 0 signers as an argument (won't be checked in compile time)
I don't really like it, so will keep current implementation